src/main/java/com/puppycrawl/tools/checkstyle/checks/naming/LocalVariableNameCheck.java

```diff
@@ -52,6 +52,25 @@
  * <pre>
  * &lt;module name="LocalVariableName"/&gt;
  * </pre>
+ * <p>Code Example:</p>
+ * <pre>
+ * class MyClass {
+ *   void MyMethod() {
+ *     for (int var = 1; var &lt; 10; var++) {} // OK
+ *     for (int VAR = 1; VAR &lt; 10; VAR++) {} // violation, name 'VAR' must match
+ *                                           // pattern '^[a-z][a-zA-Z0-9]*$'
+ *     for (int i = 1; i &lt; 10; i++) {} // OK
+ *   }
+ * }
+ * </pre>
+ * <pre>
+ * class MyClass {
+ *   void MyMethod() {
+ *     for (int var_1 = 0; var_1 &lt; 10; var_1++) {} // violation, name 'var_1' must match
```

Comment:
Sorry if my original wording was misleading, please merge these 2 methods together so it is just 1 big example. I didn't mean 2 complete isolated examples.
Example:
````
 * class MyClass {
 *   void MyMethod() {
 *     for (int var = 1; var &lt; 10; var++) {} // OK
 *     for (int VAR = 1; VAR &lt; 10; VAR++) {} // violation, name 'VAR' must match
 *                                           // pattern '^[a-z][a-zA-Z0-9]*$'
 *     for (int i = 1; i &lt; 10; i++) {} // OK
 *     for (int var_1 = 0; var_1 &lt; 10; var_1++) {} // violation, name 'var_1' must match
 *                                                    // pattern '^[a-z][a-zA-Z0-9]*$'
 *   }
 * }
````

Reply:
Done.